styles/users/SettingsEdit.module.css

```diff
@@ -0,0 +1,55 @@
+.content {
```

Comment:
separate out the stylesheet between components and the page
(take a look at the other components to see how they're structured)

Reply:
if the multiple components have the same styles you can import the same stylesheet to multiple components
feel free to ask me for clarification